src/skill.cpp

```diff
@@ -320,6 +321,12 @@ bool SkillLevel::isRusty() const
            _knowledgeExperience - _exercise >= ( _level + 1 ) * ( _level + 1 ) * 10;
 }
 
+bool SkillLevel::rust_grace_period() const
```

Comment:
Does this need a separate function? Seems that you could simply put this code near the top of rust() below.
The name of the function also doesn't match the return value (it doesn't return a "period").

Reply:
That's fair. I removed this function.